main.py

```diff
@@ -225,10 +228,14 @@ def request_access(self, update: Update, context: CallbackContext, access_level:
             self.command_unknown(update, context)
             raise PermissionError(f"UserId {update.effective_user.id}, handle {update.effective_user.name} is attempting level {access_level} command without permission.")
 
-    def device_actuator(self, update: Update, context: CallbackContext, command, message: str) -> None:
+    def device_actuator(self, update: Update, context: CallbackContext, command: Union[str, list], bot_command: str, message: str) -> None:
         self.request_access(update, context, AccessLevel.DEVICE)
         device = self.get_device(update, context)
         if device:
+            supported_commands = device['supported_commands']
+            if not bot_command in supported_commands:
```

Comment:
Another way to write it that's more English is `if bot_command not in supported_commands:`

Reply:
ok